Runtime/Scripts/Export/GltfWriter.cs

```diff
@@ -1582,6 +1673,40 @@ NativeArray<byte> outputStream
             return job;
         }
 
+        static async Task ConvertSkinningAttributes(
+            AttributeData indicesAttrData,
+            int byteStride,
+            int outputByteStride,
+            int vertexCount,
+            NativeArray<byte> input,
+            NativeArray<byte> output
+        )
+        {
+            var job = CreateConvertSkinningAttributesJob(indicesAttrData, byteStride, outputByteStride, vertexCount, input, output);
+            while (!job.IsCompleted) {
+                await Task.Yield();
+            }
+            job.Complete(); // TODO: Wait until thread is finished

```

Comment:
TODO comment is not required, since the job is awaited for in the while loop above